data-raw/england-ltla-secondary-care.R

```diff
@@ -160,11 +163,66 @@ bed_occupancy_ltla <-
     .after = ltla21_code
   )
 
+# ---- A&E attendances over 4 hours ----
+# Data is collected for all A&E types including walk-in centres and
+# minor injury units. Only acute hospitals are mapped to LTLAs; therefore
+# A&E types are mapped to acute hospitals.
+
+attendances_4hours_trust <- england_trust_accidents_emergency |>
+  mutate(date = parse_date_time(date, orders = "B Y")) |>
+  filter(date >= ymd("2023-01-01"), date < ymd("2023-04-01")) |>
+  select(nhs_trust22_code, attendances_over_4hours, total_attendances) |>
+  group_by(nhs_trust22_code) |>
+  summarise(
+    number_mean = mean(attendances_over_4hours, na.rm = TRUE),
+    total_mean = mean(total_attendances, na.rm = TRUE)
```

Comment:
the col names `number_mean` and `total_mean` aren't clear what they represent. Can we keep the original col names and adjust append the suffix `_mean` to the end instead?